### PR TITLE
Add types for pseudo-audio-param

### PR DESCRIPTION
**Pull request by kirbysayshi**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/pseudo-audio-param/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// Type definitions for pseudo-audio-param 1.3
+// Project: https://github.com/mohayonao/pseudo-audio-param/
+// Definitions by: Drew Petersen <https://github.com/kirbysayshi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "pseudo-audio-param" {
```
Comment (sandersn) [inline types/pseudo-audio-param/index.d.ts]: this is the old UMD pattern. The new style uses the syntax `export as namespace PseudoAudioParam`. So the code would look like this:

```ts
export = PseudoAudioParam;
export as namespace PseudoAudioParam;
// rest of code is the same 
interface PseudoAudioParamEvent {
   // etc etc etc...
```

**Review by sandersn — CHANGES_REQUESTED** [review]

File: types/pseudo-audio-param/index.d.ts (hunk 1 above)
Comment (kirbysayshi) [reply types/pseudo-audio-param/index.d.ts]: Ok, I made the changes requested! Thanks for the sample code. I find UMD typings one of the most difficult to get understand and get right.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/pseudo-audio-param/tslint.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "no-single-declare-module": false,
```
[the remark this answers is not in the document]
Comment (kirbysayshi) [reply types/pseudo-audio-param/tslint.json]: Thank you, I didn't know and was following existing code. Thanks for telling me! Hopefully fixed.

File: types/pseudo-audio-param/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    /**
+     * Return scheduled value at time
+     */
+    getValueAtTime(time: number): number;
+
+    /**
+     * Apply scheduled methods to the provided audioParam. If reset is `true`,
+     * cancel all events of AudioParam before applying
+     */
+    applyTo(audioParam: AudioParam, reset: boolean): PseudoAudioParam;
```
[the remark this answers is not in the document]
Comment (kirbysayshi) [reply types/pseudo-audio-param/index.d.ts]: That seems reasonable! Fixed.

**Review by sheetalkamat — APPROVED** [review]

**Pull request merged**